Section3-PeerReview-Writeup/README.md

```diff
@@ -0,0 +1,65 @@
+# Section3-PeerReview-WriteUp 
+
+## Introduction 
+The goal of today's section is to learn about peer review in regards to our class
+and code review in industry. In this section, you will be introduced to Aropa, our 
+peer review tool, and write your first two peer reviews. 
+
+Start the section by reading about the benefits of Code Review in industry by 
+following the link below. 
+
+https://blog.fullstory.com/what-we-learned-from-google-code-reviews-arent-just-for-catching-bugs/
+
+
+
+## Setup for Peer Review 
+You will need to go to the Aropa website, click on the U of A logo or select
+the University of Arizona on the dropdown menu, and then log in using your UA
+netid and the class code 386198. The website is linked below and the same 
+instructions are posted on Piazza. 
+
+https://aropa2.gla.ac.uk/aropa/aropa.php
+
+You will be prompted to change your password. Choose a secure password that 
+you will remember as you will use this same account for peer reviews in 
+the future. 
+
+## The Assignment
+Overview : Upload your PA2 code for review and write two peer reviews for PA2. 
+
+### Upload Process 
```

Comment:
We are going to have to ask them to do this ahead of time.  Let's discuss Thursday night.